app/src/main/java/eu/faircode/netguard/ApplicationEx.java

```diff
@@ -30,7 +30,7 @@
     @Override
     public void onCreate() {
         super.onCreate();
-        Log.i(TAG, "Create version=" + Util.getSelfVersionName(this));
+        Log.i(TAG, "Create version=" + Util.getSelfVersionName(this) + "(" + Util.getSelfVersionCode(this) + ")");
```

Comment:
Good, but please separate the version name and code by a slash and add this in ActivityMain as well.